No menu entry if weight <= 0

knarrff opened the pull request:
This patch allows hosting of pages without a menu entry. At the moment, all pages would create a menu entry, even those without a weight set.

knarrff committed:
No menu entry if weight <= 0

This patch allows hosting of pages without a menu entry. At the moment,
all pages would create a menu entry, even those without a weight set.

fiveop commented on _includes/navbar.html:
```diff
@@ -31,7 +31,7 @@
                 {% for node in pg %}
                     <!-- Neglect imprint.html and index.html in either language -->
-                    {% unless node.path contains "imprint" or node.path contains "index" %}
+                    {% unless node.path contains "imprint" or node.path contains "index" or node.weight <= 0 %}
```
What is the weight with non explicitly provided? And if it is zero, couldn’t we get rid of the first two special cases? And if it isn’t, why don’t we set it explicitly to zero for imprint and index and remove the special cases?

knarrff replied:
Good point. Will need to test this.

knarrff committed:
requested change: remove special cases

This removes the special cases of imprint and index. Those did not have
weight defined, while all other had. From this on, all pages without
weight defined will not been shown in the menu, not only those two.
